### ISSUE TITLE
500 Error code appears randomly on pages 

### ISSUE THREAD

**Issue by ReduktorSpalin** [zendframework/documentation#11]
### TL;DR 

500 Internal Server Error status code on Zend Framework documentation pages. It seems as if the cause is common to all pages (error rate might depend on hosting limitations).

### Story and examples

I frequently use old ZF2 documentation pages to see changes in framework and for some legacy projects. But for a month or longer I see __500 Internal Server Error__ message and __Oops!__ page showing randomly.

#### Example
![image](https://user-images.githubusercontent.com/46669552/54588759-1dc15b80-4a24-11e9-8b92-8408c6ded0c6.png)

![image](https://user-images.githubusercontent.com/46669552/54588773-25810000-4a24-11e9-9ba6-d82e050e473c.png)

**Comment by froschdesign** [comment]
This problem is tracking on the website repository: https://github.com/zendframework/zf3-web/issues/114
